package.json

```diff
@@ -6,7 +6,9 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "bootstrap": "^4.5.0",
     "react": "^16.13.1",
+    "react-bootstrap": "^1.2.1",
```

Comment:
Have you asked anyone before using bootstrap and not material-ui ?

Reply:
Its fine to use the react-strap @aayush-1999. Is there any other drawbacks which u r thinking of